MN/Algoritmi/Metode-Iterative/Ecuatii-Neliniare/bisection_method.m

```diff
@@ -0,0 +1,32 @@
+function [x i] = BisectionMethod(f, a, b, tol, max_iter)
```

Comment:
- virgula dupa x, am uitat sa-ti zic => [x, i]
- explica in niste comentarii cine sunt f, a, b, tol, max_iter (poate si x, i) --- te poti uita la MetodaTangentei